templates/web/pages/product_edit/product_edit_form_display.tt.html

```diff
@@ -207,7 +207,11 @@ <h1>[% title %]</h1>
                                     <td>
                                         <!--label starts-->
                                         [% IF nutriment.name.defined %]
-                                            <label class="nutriment_label" for="nutriment_[% nutriment.enid %]">[% nutriment.prefix %][% nutriment.name %]</label>
+                                        [% IF nutriment.nid == 'fat' || nutriment.nid == 'saturated-fat' || nutriment.nid == 'salt'|| nutriment.nid =='sugars' %]
```

Comment:
> Hi @MonalikaPatnaik , thanks for the PR!
> 
> If we put a `*`, we need to explicitly tell users what this `*` means, otherwise there is no way to guess. Could you add a string after the table that states "`*`: used to compute the Nutri-Score" or similar?
> 
> That string would need to be translatable, so you would need to use the lang() function, and add it in the translation files: po/common/common.pot + po/common/en.po

Thanks for the review..will make necessary changes :)